rotkehlchen/chain/ethereum/modules/yearn/vaultsv2.py

```diff
@@ -45,21 +37,12 @@ def __init__(
             premium: Optional['Premium'],  # pylint: disable=unused-argument
             msg_aggregator: MessagesAggregator,
     ) -> None:
+        self.premium = premium
```

Comment:
Why add premium here? Where is it used?

Reply:
Thought that it was used in the superclass but is not the case. Removed it